lib/macho/view.rb

```diff
@@ -0,0 +1,23 @@
+module MachO
```

Comment:
http://editorconfig.org/ and an `.editorconfig` file in the repository root to the rescue?